modules/persistent-storage-local-create-cr.adoc

```diff
@@ -56,7 +56,12 @@ spec:
 <4> The volume mode, either `Filesystem` or `Block`, that defines the type of local volumes.
 <5> The file system that is created when the local volume is mounted for the first time.
 <6> The path containing a list of local storage devices to choose from.
-<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/disk/by-id/wwn`. PVs are created for these local disks when the provisioner is deployed successfully.
+<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource id, such as `/dev/disk/by-id/wwn`. PVs are created for these local disks when the provisioner is deployed successfully.
++
+[NOTE]
+====
+Local Storage Operator does not  support `LocalVolume` resource by path, such as `/dev/disk/by-path/wwn`.
+====
```

Comment:
Would like to add "does not  support `LocalVolume` resource by path or by partuuid" from https://bugzilla.redhat.com/show_bug.cgi?id=2052618#c3

Reply:
Fixed!